zng/docs/zson.md

```diff
@@ -0,0 +1,630 @@
+# ZSON - ZNG Structured Object-record Notation
+
+> ### DRAFT 11/30/20
+> ### Note: This specification is in BETA development.
+> We plan to have a final form established by Spring 2021.
+>
+> ZSON is intended as a literal superset of JSON syntax (not semantics).
```

Comment:
It's a superset of NDJSON 🤓

Reply:
It's a superset of both JSON and NDJSON.  I will add NDJSON.  (zq won't handle top-level values that aren't records but the marshaler will handle any value.)

Reply:
Yep, JSON ⊂ NDJSON ⊂ ZSON